tests/foreman/api/test_contentview.py

```diff
@@ -693,6 +700,92 @@ def test_positive_publish_with_puppet_multiple(self):
         for cvv in content_view.read().version:
             self.assertEqual(len(cvv.read().puppet_module), 1)
 
+    @tier2
+    def test_positive_publish_with_swid_tags(self):
+        """Verify SWID tags content file should exist in publish content view
+        version location
+
+        :id: 25f10e99-ceef-4543-9b9c-fc21690c088b
+
+        :steps:
+            1. create product and repository with custom contents having swid tags
+            2. sync the repository
+            3. create the content view
+            4. publish the content-view
+            5. ssh into Satellite
+            6. verify SWID tags content file exist in publish content view version location
+
+        :expectedresults: SWID tags content file should exist in publish content view
+            version location
+
+        :CaseAutomation: Automated
```

Comment:
You have the same definition on module level, so there is no need to repeat it here (but repeating it isn't wrong!).